src/sagemaker/image_uri_config/debugger.json

```diff
@@ -24,7 +24,8 @@
                 "us-east-1": "503895931360",
                 "us-east-2": "915447279597",
                 "us-west-1": "685455198987",
-                "us-west-2": "895741380848"
+                "us-west-2": "895741380848",
+                "us-gov-west-1": "515509971035"
```

Comment:
Do we also need to remove us-gov-west-1 from DEBUGGER_UNSUPPORTED_REGIONS? https://github.com/aws/sagemaker-python-sdk-staging/blob/master/src/sagemaker/fw_utils.py#L51

Reply:
https://github.com/aws/sagemaker-python-sdk/blob/5f52f95483a310058b193d408034886c353c2685/src/sagemaker/fw_utils.py#L51